tests/Integration/ResourceStatusSystem/Test_ResourceManagement.py

```diff
@@ -8,281 +8,50 @@
-
-  res = rsClient.deleteSpaceTokenOccupancyCache('TestName1234')
-  assert res['OK'] == True
-
-  res = rsClient.selectSpaceTokenOccupancyCache('TestName1234')
-  assert res['OK'] == True
-  assert not res['Value']
-
-
-  # TEST addOrModifyUserRegistryCache
-  # ...............................................................................
-
-  res = rsClient.addOrModifyUserRegistryCache('TestName1234')
-  assert res['OK'] == True
-
-  res = rsClient.selectUserRegistryCache('TestName1234')
-  assert res['OK'] == True
-  assert res['Value'][0][0] == 'TestName1234'
-
-
-  # TEST deleteUserRegistryCache
-  # ...............................................................................
-
-  res = rsClient.deleteUserRegistryCache('TestName1234')
-  assert res['OK'] == True
-
-  res = rsClient.selectUserRegistryCache('TestName1234')
-  assert res['OK'] == True
-  assert not res['Value']
-
-
-  # TEST addOrModifyVOBOXCache
-  # ...............................................................................
-
-  res = rsClient.addOrModifyVOBOXCache('TestName1234')
-  assert res['OK'] == True
-
-  res = rsClient.selectVOBOXCache('TestName1234')
-  assert res['OK'] == True
-  assert res['Value'][0][2] == 'TestName1234'
-
+  rsClient.deleteAccountingCache('TestName12345')
 
-  # TEST deleteVOBOXCache
+  # TEST addOrModifyAccountingCache
   # ...............................................................................
 
-  res = rsClient.deleteVOBOXCache('TestName1234')
+  res = rsClient.addOrModifyAccountingCache('TestName12345', 'plotType', 'plotName', 'result', dateEffective, lastCheckTime)
   assert res['OK'] == True
 
-  res = rsClient.selectVOBOXCache('TestName1234')
+  res = rsClient.selectAccountingCache('TestName12345')
   assert res['OK'] == True
-  assert not res['Value']
-
-
-  # TEST insertErrorReportBuffer
-  # ...............................................................................
+  #check if the name that we got is equal to the previously added 'TestName12345'
+  assert res['Value'][0][0] == 'TestName12345'
 
-  res = rsClient.insertErrorReportBuffer('TestName1234')
+  res = rsClient.addOrModifyAccountingCache('TestName12345', 'plotType', 'plotName', 'changedresult', dateEffective, lastCheckTime)
   assert res['OK'] == True
 
-  res = rsClient.selectErrorReportBuffer('TestName1234')
-  assert res['OK'] == True
-  assert res['Value'][0][1] == 'TestName1234'
+  res = rsClient.selectAccountingCache('TestName12345')
+  #check if the result has changed
+  assert res['Value'][0][3] == 'changedresult'
 
 
-  # TEST deleteErrorReportBuffer
+  # TEST deleteAccountingCache
   # ...............................................................................
-
-  res = rsClient.deleteErrorReportBuffer('TestName1234')
+  res = rsClient.deleteAccountingCache('TestName12345')
   assert res['OK'] == True
 
-  res = rsClient.selectErrorReportBuffer('TestName1234')
+  res = rsClient.selectAccountingCache('TestName12345')
```

Comment:
Theoretically all the methods are checked because the client calls a generic method from the DB class which is either `select`, `delete` or `addOrModify` so all the methods should work the same. Regarding the tables I guess I need to extend it to cover more tables.